apollo-router/src/warp_http_server_factory.rs

```diff
@@ -345,7 +345,7 @@ where
     name = "graphql_request",
     fields(
         query = %request.query.clone().unwrap_or_default(),
-        operation_name = %request.operation_name.clone().unwrap_or_else(|| "-".to_string()),
+        operation_name = %request.operation_name.clone().unwrap_or_else(|| "".to_string()),
```

Comment:
That's going to break the studio usage though. If you look at [stats_report_key() in apollo_open](https://github.com/apollographql/router/blob/9537ef271ac76e449220c0ed5abe1a8632955673/apollo-router/src/apollo_telemetry.rs#L388) you'll note that we assume the default name for an operation there is "-" which is what we want for stats reporting.

To keep things working, you'll need to modify `stats_report_key()` to check if `op_name` is `""` and, if it is, then update that logic to substitute `"-"` as the `op_name`. The comment there will need to be updated as well.